Update network-topology-connectivity.md to reflect app gateway should not be used with ingress controller (#1372)
MicrosoftDocs/cloud-adoption-framework: docs/scenarios/app-platform/azure-red-hat-openshift/network-topology-connectivity.md

- [Ingress controllers](https://docs.openshift.com/container-platform/latest/networking/ingress-operator.html) provide application-level routing with only a slight increase in complexity.
- Azure Red Hat OpenShift creates a generic DNS entry that simplifies access to exposed applications in the cluster. An example DNS entry is `*.apps.<cluster-ID>.<region>.aroapp.io`. It's useful for a private cluster to route traffic for the application.
- Azure Red Hat OpenShift offers a built-in [Ingress controller and routes](https://docs.openshift.com/container-platform/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-ingress-controller.html).

> - You can use Azure Red Hat OpenShift ingress with [Azure Front Door](/azure/openshift/howto-secure-openshift-with-front-door). **Note** that using Application Gateway with Azure Red Hat Openshift ingress controllers defined as backend settings is **NOT** supported due to service chaining.  Follow the design guidance in this article.

Reviewer: ```suggestion
- You can use Azure Red Hat OpenShift ingress with [Azure Front Door](/azure/openshift/howto-secure-openshift-with-front-door). Using Azure Application Gateway with Azure Red Hat OpenShift ingress controllers defined in backend pools is not supported due to service chaining. Follow the design guidance in this article.
```

> - You can use Azure Red Hat OpenShift ingress with [Azure Front Door](/azure/openshift/howto-secure-openshift-with-front-door). **Note** that using Application Gateway with Azure Red Hat Openshift ingress controllers defined as backend settings is **NOT** supported due to service chaining.  Follow the design guidance in this article.

Reviewer: Do you have a link to include here that defines _service chaining_ so a customer understands why that is a bad thing?

Reviewer 2: @vallentyne Can you respond to the comment above?

@MicrosoftDocs/public-repo-pr-review-team
